rilldata/rill-developer
# Conditionally emphasize the Add Asset buttons

## Checklist

## Summary
#### Issue addressed: 
[Notion PRD](https://www.notion.so/rilldata/Improving-Target-size-for-Add-Source-Models-Dashboard-Nav-c04af12473a642e8b87143b826a12038?d=cc146267e37e49a7945960a0920b0bd5#cbce4b2994fb454e92804f30ae02c8d1)

#### Details:
This PR implements the "Suggested Actions" component of the PRD.

This PR follows #2538, which should be reviewed first.

## Steps to Verify
1. Create a new project
2. Add sources, models, and dashboards
3. Verify that the Add Asset buttons are conditionally bold according to the spec
